src/situations/commun/vues/bouton.js

```diff
@@ -5,13 +5,28 @@ export default class VueBouton {
     this.click = click;
   }
 
+  ajouteUneEtiquette (etiquette, aGauche = false) {
+    this.etiquette = etiquette;
+    this.aGauche = aGauche;
+  }
+
   affiche (pointInsertion, $) {
-    this.$bouton = $(`<a class="${this.classe}"><img src="${this.image}"></a>`);
-    this.$bouton.on('click', this.click);
-    $(pointInsertion).append(this.$bouton);
+    const $bouton = $(`<a class="${this.classe}"><img src="${this.image}"></a>`);
+    $bouton.on('click', this.click);
+    this.$element = $bouton;
+    if (this.etiquette && this.etiquette.length > 0) {
```

Comment:
A quoi sert le `this.etiquette.length > 0` ?

Reply:
C'est un reste d'une implémentation précédente. Je vais l'enlever.

Reply:
fait